### PR TITLE
Add files via upload

### LINKED ISSUE #127: Theme Switcher
I want to create theme switcher using HTML, CSS and JavaScript.

### PR DESCRIPTION
**Pull request by vimarsh11**
I created a Theme-Switcher with the help of HTML, CSS, JS. There are three file which are uploaded i.e
1)HTML
2)CSS
3)JavaScript.

fixed #127 
Screenshot
Screenshot 2023-01-13 020650

### CODE REVIEW

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
Add a README.md file with the explanation of your project, the tech stack used, and a screenshot of the project.

**Comment by vimarsh11** [comment]
@Vanshiii203 I added the readme file

File: Projects/Theme Switcher/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+3)JavaScript.
+which switch the theme light and dark modes
+</pre>
+<h2>TechStack</h2>
+<pre>
+The HyperText Markup Language or HTML is the standard markup language for documents designed to be displayed in a web browser.
+It can be assisted by technologies such as Cascading Style Sheets (CSS) and scripting languages such as JavaScript.
+</pre>
+<h2>Screensot</h2>
+<img src="screenshot"/>
```
Comment (Vanshiii203) [inline Projects/Theme Switcher/README.md:14]: The image is not visible. Use- ![screenshot] (screenshot.jpg)

Comment (vimarsh11) [reply Projects/Theme Switcher/README.md]: @Vanshiii203 screenshot updated

**Review by Vanshiii203 — CHANGES_REQUESTED** [review]
The screenshot is not visible in the readme.md file

**Comment by vimarsh11** [comment]
From my side screenshot is visible. @Vanshiii203 @harshit-paneri @LiQuiD-404 
Screenshot 2023-01-18 132127

**Review by Vanshiii203 — APPROVED** [review]

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**